src/engine/UIManager.cxx

```diff
@@ -105,6 +105,7 @@ void UIManager::init()
     auto &elements = uiLayout["UiElements"][it.key()];
 
     // parse UiElements
+    auto &settings = Settings::instance();
```

Comment:
I don't think this is necessary. Settings is a singleton and is accessed just by using `Settings::instance()` throughout this and other files.

Reply:
ok